statsd/statsd.go

```diff
@@ -19,33 +19,60 @@ import (
 	"golang.org/x/net/context"
 )
 
+// DefaultBackends is the list of default backends.
 var DefaultBackends = []string{"graphite"}
+
+// DefaultMaxReaders is the default number of socket reading goroutines.
 var DefaultMaxReaders = runtime.NumCPU()
+
+// DefaultMaxMessengers is the default number of message processing goroutines.
 var DefaultMaxMessengers = runtime.NumCPU() * 8
+
+// DefaultMaxWorkers is the default number of goroutines that aggregate metrics.
 var DefaultMaxWorkers = runtime.NumCPU() * 8 * 8
+
+// DefaultPercentThreshold is the default list of applied percentiles.
 var DefaultPercentThreshold = []string{"90"}
+
+// DefaultTags is the default list of additional tags.
 var DefaultTags = []string{""}
 
 const (
+	// DefaultExpiryInterval is the default expiry interval for metrics.
 	DefaultExpiryInterval = 5 * time.Minute
-	DefaultFlushInterval  = 1 * time.Second
-	DefaultMetricsAddr    = ":8125"
+	// DefaultFlushInterval is the default metrics flush interval.
+	DefaultFlushInterval = 1 * time.Second
+	// DefaultMetricsAddr is the default address on which to listen for metrics.
+	DefaultMetricsAddr = ":8125"
 )
 
 const (
-	ParamBackends         = "backends"
-	ParamConsoleAddr      = "console-addr"
-	ParamCloudProvider    = "cloud-provider"
-	ParamDefaultTags      = "default-tags"
-	ParamExpiryInterval   = "expiry-interval"
-	ParamFlushInterval    = "flush-interval"
-	ParamMaxReaders       = "max-readers"
-	ParamMaxMessengers    = "max-messengers"
-	ParamMaxWorkers       = "max-workers"
-	ParamMetricsAddr      = "metrics-addr"
-	ParamNamespace        = "namespace"
+	// ParamBackends is the name of parameter with backends.
```

Comment:
It wasn't magic strings, it is the name of each flag so it makes sense to have it sitting, er with the flag :P. Originally it was only defined in the flags, but maybe you changed that in your previous refactoring hence it needed to put it as a constant? Otherwise imho it's just unnecessary indirection.